### ISSUE TITLE
Can't see COVID-19 Exposure Notifications send with Android.

### ISSUE THREAD

**Issue by zeno4ever** [citruz/beacontools#48]
I use the beacontools for my project, ["CoronaTeller"](https://github.com/zeno4ever/CoronaTeller/issues/3) (dutch for Corona counter)

Testing :

```bash
  $ bluetoothctl
  [bluetoothctl]scan.transport le
  [bluetoothctl]scan.uuids 0xfd6f
  [bluetoothctl]scan on
```

Gives all beacons including the Android phones. Running the beacontoools scan tool (without any packet filtering) gives only my iPhone, not an Android phone.

**Comment by citruz** [comment]
Unfortunately I don't have an Android phone to test that. Can you provide me with a raw dump of the ble packet?
You should be able do get it using:
```bash
sudo hcitool lescan --duplicates
# (in another terminal)
sudo hcidump --raw | grep --line-buffered  "6F FD" -A 1 -B 1
```

**Comment by zeno4ever** [comment]
-- Android
> 04 3E 28 02 01 03 01 BB 7E B5 2B 86 79 1C 03 03 6F FD 17 16 
  6F FD 2C FB 0D E0 2B 33 D2 0C 5C 27 61 12 38 E2 D1 07 42 B5 
  6E E5 B8 
-- iPhone
> 04 3E 2B 02 01 03 01 08 E6 AE 33 0B 3F 1F 02 01 1A 03 03 6F 
  FD 17 16 6F FD E9 32 E8 B0 68 8D FA EC 00 62 B7 D6 D3 5E EF 
  B5 EE AA 91 AC BA

**Comment by citruz** [comment]
I just published 2.0.2 in which I made packet pre-filtering more resilient against slightly different packet layouts. This should the issue.

**Comment by zeno4ever** [comment]
Tested, seems to work. Great work, thanks @citruz
